ivy/functional/frontends/numpy/mathematical_functions/sums_products_differences.py

```diff
@@ -26,3 +26,24 @@ def prod(
 
 
 prod.unsupported_dtypes = {"torch": ("float16",)}
+
+
+# sum
+def sum(
+        x,
```

Comment:
please follow the signature from numpy docs

from numpy:
`numpy.sum(a, axis=None, dtype=None, out=None, keepdims=<no value>, initial=<no value>, where=<no value>)`

replace initial <no value> with None, and where <no value> with the default True